src/DotLiquid/Strainer.cs

```diff
@@ -1,8 +1,10 @@
 using System;

```

Comment:
Can we add some StrainerTests? I know the code tends to test things indirectly through the "system level" functions (e.g. Parse, Render), but it would be good to have dedicated tests for some of the main classes.

Reply:
This could be raised as an issue (or added to #585) to be done later.

Reply:
Agree to defer, although there is definitely opportunity to add unit tests to test the new functionality.